extra/mariabackup/xtrabackup.cc

```diff
@@ -1816,6 +1816,12 @@ static void print_version(void)
       my_progname, MYSQL_SERVER_VERSION, SYSTEM_TYPE, MACHINE_TYPE);
 }
 
+static void concatenate_default_groups(std::vector<const char*> &backup_load_groups, const char **default_groups)
+{
+  for( ; *default_groups; ++default_groups)
+    backup_load_groups.emplace_back(*default_groups);
```

Comment:
`push_back` is more readable and preferrable for primitive data types, so I would suggest to use as in first patch.

Reply:
I got it, Thank you very much for your reply.